lib/level10/games/game.ex

```diff
@@ -5,46 +5,144 @@ defmodule Level10.Games.Game do
   game's state, and then stored on the server to be updated or to serve data
   down to clients.
   """
+  # alias Level10.Games.{Card, GameRegistry, Player}
+  alias Level10.Games.Player
 
-  alias Level10.Games.{Card, Player}
-
+  @type join_code :: <<_::32>>
```

Comment:
Is it bad that I don't actually know what this type is saying? Is that saying it's a 32-bit binary? I'm assuming that's probably the case since utf-16 is 4 bytes. Is there a reason to do it this way rather than just say `String.t()`?

Reply:
Yes, that's what it's saying
I just thought it would be more specific, but I can change that